frontend/server/src/Controllers/Course.php

```diff
@@ -2363,41 +2363,42 @@ public static function getCourseListDetailsForSamrty(\OmegaUp\Request $r): array
         $coursesTypes = ['public', 'student', 'admin'];
 
         $filteredCourses = [];
-        foreach ($coursesTypes as $index => $courseType) {
-            $filteredCourses[$index] = [
-                'accessMode' => $courseType,
+        foreach ($coursesTypes as $courseType) {
+            $filteredCourses[$courseType] = [
                 'filteredCourses' => [
-                    ['courses' => [], 'timeType' => 'current'],
-                    ['courses' => [], 'timeType' => 'past'],
+                    'current' => ['timeType' => 'current'],
```

Comment:
esto estaba bien con el índice `'courses'`:

```suggestion
                    'current' => [
                        'courses' => [],
                        'timeType' => 'current',
                    ],
```

Reply:
Listo